src/Core/src/Platform/Windows/ControlExtensions.cs

```diff
@@ -21,6 +21,9 @@ public static void UpdateFont(this Control platformControl, Font font, IFontMana
 			platformControl.IsTextScaleFactorEnabled = font.AutoScalingEnabled;
 		}
 
+		public static void UpdateIsEnabled(this Control platformControl, IView view) =>
```

Comment:
I think the only place using this method is ViewExtensions.cs below, we can probably just inline this and avoid adding more API.